androidApp/src/main/java/com/mbta/tid/mbta_app/android/more/MoreLink.kt

```
@@ -14,45 +17,43 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.colorResource
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.LinkAnnotation
-import androidx.compose.ui.text.buildAnnotatedString
-import androidx.compose.ui.text.withLink
 import androidx.compose.ui.unit.dp
 import com.mbta.tid.mbta_app.android.R
 
 @Composable
 fun MoreLink(label: String, url: String, note: String? = null, isKey: Boolean = false) {
+    val context = LocalContext.current
 
     Row(
         modifier =
-            Modifier.background(
-                color =
-                    if (isKey) {
-                        colorResource(R.color.key)
-                    } else {
-                        colorResource(R.color.fill3)
+            Modifier.clickable {
+                    val webpage: Uri = Uri.parse(url)
+                    val intent = Intent(Intent.ACTION_VIEW, webpage)
+                    if (intent.resolveActivity(context.packageManager) != null) {
+                        context.startActivity(intent)
                     }
```

From a bit more of an in-depth investigation, it looks like `resolveActivity` doesn't just check that the intent can be handled but returns the activity that will handle it, and we do not need to know which web browser the user has. It looks like if the user has no web browser installed `startActivity(intent)` will throw an `ActivityNotFoundException`, so it may be sufficient to simply `catch` and discard that exception instead of changing the manifest so we can `resolveActivity`.

---

Yep, that worked! Nice catch.